src/uu/factor/src/table.rs

```diff
@@ -43,6 +41,34 @@ pub(crate) fn factor(mut num: u64, mut factors: Factors) -> (Factors, u64) {
             }
         }
     }
+}
+
+pub const CHUNK_SIZE: usize = 8;
+pub fn factor_chunk(n_s: &mut [u64; CHUNK_SIZE], f_s: &mut [Factors; CHUNK_SIZE]) {
```

Comment:
this functions seems to be untested, is that expected?

Reply:
Yes.  As I said, the tests I'd want to write for it requires implementing `quickcheck::Arbitrary` for `[u64; CHUNK_SIZE]`, which I cannot do here (array types are foreign, and so is the trait) ; there is a quickcheck PR which implements that, but we won't be able to pull it in for a while (due to the use of `const` generics)

I guess I should write an alternative that takes a single `u64`, and uses it as a seed to generate the actual test data; it will break displaying failing testcases (showing the seed instead) and testcase minimization (aka shrinking) but it's better than no test at all.

Reply:
Done